pom.xml

```diff
@@ -16,25 +16,22 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.build.timestamp.format>yyyy-MM-dd HH:mm:ss</maven.build.timestamp.format>
     <timestamp>${maven.build.timestamp}</timestamp>
-
     <maven.compiler.parameters>true</maven.compiler.parameters>
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
-
     <!-- Plug-in versions -->
     <frontend.maven.plugin.version>1.10.0</frontend.maven.plugin.version>
     <maven.clean.plugin.version>3.1.0</maven.clean.plugin.version>
     <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
     <maven.jar.plugin.version>3.2.0</maven.jar.plugin.version>
     <maven.resources.plugin.version>3.1.0</maven.resources.plugin.version>
-    <maven.surefire.plugin.version>2.22.1</maven.surefire.plugin.version>
-
-    <quarkus.plugin.version>1.8.1.Final</quarkus.plugin.version>
+    <maven.surefire.plugin.version>3.0.0-M5</maven.surefire.plugin.version>
+    <quarkus.plugin.version>1.10.2.Final</quarkus.plugin.version>
     <quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
-    <quarkus.platform.version>1.8.1.Final</quarkus.platform.version>
+    <quarkus.platform.version>1.10.2.Final</quarkus.platform.version>
     <testcontainers.version>1.14.3</testcontainers.version>
-    <debezium.base.version>1.3.0.Beta2</debezium.base.version>
+    <debezium.base.version>1.4.0.CR1</debezium.base.version>
```

Comment:
We just decided we'll wait for tomorrow's v1.4.0.Final release and use that released version here.